src/main/java/org/mule/module/apikit/CharsetUtils.java

```diff
@@ -6,51 +6,39 @@
  */
 package org.mule.module.apikit;
 
-import org.mule.module.apikit.helpers.EventHelper;
+import org.mule.module.apikit.helpers.AttributesHelper;
 import org.mule.runtime.api.metadata.MediaType;
 import org.mule.runtime.api.metadata.TypedValue;
-import org.raml.parser.utils.StreamUtils;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.nio.charset.Charset;
 import java.util.Optional;
-
-import static org.mule.module.apikit.helpers.PayloadHelper.getPayloadAsByteArray;
+import org.mule.runtime.api.util.MultiMap;
 
 public class CharsetUtils {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(CharsetUtils.class);
-
+  private CharsetUtils() {}
 
-  private static String normalizeCharset(String encoding) {
```

Comment:
Why do we remove normalizeCharset? the behavior will change and things may get broken